kobo/apps/openrosa/apps/main/migrations/0015_drop_old_restservice_tables.py

```diff
@@ -0,0 +1,77 @@
+# Generated by Django 4.2.11 on 2024-07-31 15:59
+
+from django.db import migrations, connections
+from django.conf import settings
+
+
+KC_REST_SERVICES_TABLES = [
+    'restservice_restservice',
+]
+
+
+def get_operations():
+    if settings.TESTING or settings.SKIP_HEAVY_MIGRATIONS:
+        # Skip this migration if running in test environment or because we want
+        # to voluntarily skip it.
+        return []
+
+    tables = KC_REST_SERVICES_TABLES
+    operations = []
+
+    sql = """
```

Comment:
answer: to drop the constraints and indexes before dropping the table itself, and because the model definitions don't exactly match the database state